server/sonar-server/src/main/java/org/sonar/server/organization/OrganizationCreationImpl.java

```diff
@@ -186,7 +190,7 @@ private OrganizationDto insertOrganization(DbSession dbSession, NewOrganization
       .setKey(newOrganization.getKey())
       .setDescription(newOrganization.getDescription())
       .setUrl(newOrganization.getUrl())
-      // TODO .setDefaultQualityGateUuid("" + qualityGateFinder.getBuiltInQualityGate(dbSession).getId())
+      .setDefaultQualityGateUuid(qualityGateFinder.getBuiltInQualityGate(dbSession).getUuid())
```

Comment:
This update should not have been done in another commit, ie "Set built-in QG as default of organization, where you have set the ORG#DEFAULT_QG column as nullable

Reply:
Moreoever, it doesn't work, as you forgot to create an entry in ORG_QUALITY_GATES => No QGate are displayed on a new org